interface/forms/LBF/new.php

```diff
@@ -1515,7 +1515,6 @@ function warehouse_changed(sel) {
 
```

Comment:
On line 799 - Replace `container-responsive` with either `container` or `container-fluid` as this isn't a class